deeplabcut/pose_estimation_3d/triangulation.py

```diff
@@ -88,7 +89,7 @@ def triangulate(
     To analyze only a few pair of videos:
     >>> deeplabcut.triangulate(config,[['C:\\yourusername\\rig-95\\Videos\\video1-camera-1.avi','C:\\yourusername\\rig-95\\Videos\\video1-camera-2.avi'],['C:\\yourusername\\rig-95\\Videos\\video2-camera-1.avi','C:\\yourusername\\rig-95\\Videos\\video2-camera-2.avi']])
     """
-    from deeplabcut.pose_estimation_tensorflow import predict_videos
+    from deeplabcut.pose_estimation_pytorch import analyze_videos
```

Comment:
I believe this should be updated to import analyze_videos from the `compat` module. This will make the code compatible with both `PyTorch` and `TensorFlow` DeepLabCut models, as it will select which to use based on the shuffle/which type of model was trained

```suggestion
    from deeplabcut.compat import analyze_videos
```